pkg/go.mod

```diff
@@ -76,7 +76,7 @@ require (
 	github.com/pgavlin/diff v0.0.0-20230503175810-113847418e2e
 	github.com/pkg/browser v0.0.0-20210115035449-ce105d075bb4
 	github.com/pulumi/pulumi-java/pkg v0.9.2
-	github.com/pulumi/pulumi-terraform-bridge/v3 v3.43.0
+	github.com/pulumi/pulumi-terraform-bridge/v3 v3.48.1-0.20230527113951-a0f92e793f31
```

Comment:
This points to a small branch (https://github.com/pulumi/pulumi-terraform-bridge/tree/fraser/getMapping) to fix up that GetMapping now needs two parameters.

I'll ensure this gets pointed back to mainline once things are merged and propgated.